source/BaselineOfHyperspace/BaselineOfHyperspace.class.st

```diff
@@ -44,8 +45,14 @@ BaselineOfHyperspace >> setUpPackages: spec [
 
 	spec
 		package: 'Hyperspace-Model' with: [ spec requires: 'Buoy-Deployment' ];
-		group: 'Deployment' with: 'Hyperspace-Model'.
+		group: 'Deployment' with: 'Hyperspace-Model';
+		package: 'Hyperspace-Model-Tests'
+			with: [ spec requires: #('Hyperspace-Model' 'Dependent-SUnit-Extensions') ];
+		group: 'Tests' with: 'Hyperspace-Model-Tests'.
+
 	spec
-		package: 'Hyperspace-Model-Tests' with: [ spec requires: #('Hyperspace-Model') ];
-		group: 'Tests' with: 'Hyperspace-Model-Tests'
+		package: 'Hyperspace-SUnit-Model';
+		group: 'Dependent-SUnit-Extensions' with: 'Hyperspace-SUnit-Model';
+		package: 'Hyperspace-SUnit-Tests' with: [ spec requires: 'Hyperspace-SUnit-Model' ];
```

Comment:
```suggestion
		package: 'Hyperspace-SUnit-Model-Tests' with: [ spec requires: 'Hyperspace-SUnit-Model' ];
```
Standard Pharo options just adds Tests to the package under test name

Reply:
I took the name from Buoy's extensions. Should we change that one in the future?